bin/lint_localized_strings_format

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash -eu
+
+# Attempts to generate the frozen `.strings` file to gets sent to GlotPress
+# after a new release code freeze.
+#
+# If the process fails, annotates the build with the errors.
+
+# The strings generation happens via Fastlane, so we need to install the
+# Ruby gems.
+echo "--- :rubygems: Setting up Gems"
+install_gems
+
+# The strings generation also picks up files from our first party libraries,
+# some of which are installed via CocoaPods
+echo "--- :cocoapods: Setting up Pods"
+install_cocoapods
+
+echo "--- :sleuth_or_spy: Lint Apple Localized Strings Format"
+# A next step improvement is to move the logs management within the release
+# toolkit
+LOGS=logs.txt
+set +e
+set -o pipefail
+if [[ $# -eq 0 ]]; then
+  bundle exec fastlane generate_strings_file_for_glotpress skip_commit:true | tee $LOGS
+else
+  bundle exec fastlane "$@" | tee $LOGS
+fi
```

Comment:
Re: your comment about this on commit ae2b014a503ad9cb243a0f9ff5b056c8d7806dd1:

> You might notice the duplicated `| tee $LOGS` call. I considered DRYing the code by assigning `bundle exec fastlane ...` to a variable, e.g. `CMD`, and then do `$CMD | tee $LOGS`, but decided to keep the duplication because it's minimal and keeps the code, IMHO, clearer.

I found another (and more bash-idiomatic) way to avoid repeating the command in each branch: you can use `set -- …` to replace the content of the `$@` arguments list:

```suggestion
if [[ $# -eq 0 ]]; then
  # If no argument provided, update $@ with default values
  set -- generate_strings_file_for_glotpress skip_commit:true
fi
bundle exec fastlane "$@" | tee $LOGS
```

(Or alternatively a more compact version below, though I'm not sure it's worth the loss of readability):

```suggestion
# If no argument provided, update $@ with default values
[[ $# -eq 0 ]] && set -- generate_strings_file_for_glotpress skip_commit:true
bundle exec fastlane "$@" | tee $LOGS
```

Reply:
TIL `set --`. It looks odd at first, but I guess it's just because I'm don't know much idiomatic bash. It certainly achieves the goal of DRYing both `bundle exec fastlane` and `| tee $LOGS` and in a way that's neat.

Reply:
TBH I didn't know about it either, I tried `unshift` at first (thinking it was a bash command that existed, as the opposite of `shift`… but no), then I Googled a bit and learned about `set --` in the process… so TIL too 😛